app/models/support/service_performance.rb

```diff
@@ -1,4 +1,318 @@
 class Support::ServicePerformance
+  def percentage_of_devolved_schools_that_have_signed_in
+    if number_of_devolved_schools.positive?
+      (number_of_devolved_schools_that_have_signed_in * 100.0 / number_of_devolved_schools).round
+    else
+      0
+    end
+  end
+
+  def number_of_devolved_schools_that_have_signed_in
+    @number_of_devolved_schools_that_have_signed_in ||=
+      School
+      .gias_status_open
+      .that_will_order_devices
+      .joins(user_schools: :user)
+      .where('users.sign_in_count > 0')
+      .count('DISTINCT(schools.id)')
+  end
+
+  def number_of_devolved_schools
+    @number_of_devolved_schools ||=
+      School
+      .gias_status_open
+      .that_will_order_devices
+      .count
+  end
+
+  def number_of_centrally_managed_schools
+    @number_of_centrally_managed_schools ||=
+      School
+      .gias_status_open
+      .that_are_centrally_managed
+      .count
+  end
+
+  def total_devices_available
+    SchoolDeviceAllocation.std_device.sum(:allocation)
+  end
+
+  def total_devices_ordered
+    SchoolDeviceAllocation.std_device.sum(:devices_ordered)
+  end
+
+  def total_devices_remaining
+    SchoolDeviceAllocation.std_device.sum('allocation - devices_ordered')
+  end
+
+  def total_routers_available
```

Comment:
Was there anything on the card on whether this should show the number that can order vs schools in any order state?

Reply:
There was something on the card about RBs that have provided all the information, but the lists on the cards seemed more like a wish list than requirements. I have implemented the design that @fofr developed, perhaps that's a question for @fofr ?